internal/version.go

```diff
@@ -9,7 +9,7 @@ import (
 
 var (
 	// Version is the git tag that this was built from.
-	Version = "unknown"
+	Version = "v0.9.0-zeta1-4-g86b841f55a"
```

Comment:
This shouldn't be updated

Reply:
ouch, doing the same mistake again and agian ... I need to find a way to run e2e test in Golang without this :'(

Reply:
I think you can build the binary and the invocation image with make and force the `unknown` tag 🤔